SelfAndPeerEvaluations/partner_eval_quiz_creator.py

```diff
@@ -0,0 +1,161 @@
+import canvasapi
+import datetime
+import json
+from typing import Optional
+from utils import make_student_id_map
+
+
```

Comment:
```suggestion
JsonValue = Union[str, int, float, bool, list['JsonValue'], 'JsonDict']
JsonDict = dict[str, JsonValue]
```

As a part of https://github.com/mfbutner/CanvasHelpers/pull/10/files#r941853539